package.json

```diff
@@ -17,6 +17,9 @@
     "url": "https://github.com/maputnik/editor"
   },
   "author": "Lukas Martinelli",
+  "contributors": [
+    "Yuri Astrakhan <YuriAstrakhan@gmail.com>"
```

Comment:
Can you remove this. Reasoning, I don't want to start adding all the [contributors](https://github.com/maputnik/editor/graphs/contributors) to the `package.json` when it's already in [GitHub](https://github.com/maputnik/editor/graphs/contributors)